tests/source_data_validation/__init__.py

```diff
@@ -1,3 +1,13 @@
 from tests import PROJECT_ROOT
 
+SCHEMAS = PROJECT_ROOT / "schemas"
+CLUSTER_MAPPING = PROJECT_ROOT / "templates_v2" / "cluster_version_mapping.json"
+CLUSTER_MAPPING_SCHEMA = SCHEMAS / "cluster_version_mapping.schema.json"
 CONTAINER_LOG_REQUESTS = PROJECT_ROOT / "container_log_requests"
+CONTAINER_LOG_SCHEMA = SCHEMAS / "container_log.schema.json"
```

Comment:
We should not expose this as path, but rather as fixtures that expose the content. E.g. `CONTAINER_LOG_SCHEMA` is misleading, as it is not a schema but a path.

Reply:
I replaced schemas with fixtures. However, the rest of the files are sometimes used as the file names only, and sometimes as their content. Is it ok as it is, or should I create fixtures for their contents as well?

Reply:
Depends on the use. If we need to use outside tests and fixtures, they cannot be fixtures. I think the main limitation right now is how to use the schemas in the tests. If we can use some `Registry` and in each tests just pass the respective schema, then that will be the best approach. If not, then what you find most reasonable.

Reply:
Right now I pass the `Registry` as argument to `Draft202012Validator` created individually for each schema. But I haven't worked with the `referencing` package before, so let me know if there is a better way.

Reply:
It is also the first time I work with this package but I found a way to retrieve the data based on file name from the registry itself. See my comment on the other thread. Hope that simplifies things.